agent/apps/ai_guardrails/deployment/urls_additions.py

```
@@ -0,0 +1,9 @@
+"""
+URL additions for ai_guardrails
+"""
+
+from django.urls import path, include
+
+urlpatterns += [
```

Ruff (0.12.2)

Undefined name `urlpatterns`

(F821)